build/anvil/devnet/docker-bake.hcl

```diff
@@ -0,0 +1,6 @@
+target "docker-metadata-action" {}
```

Comment:
Do we need these bake files? Are they necessary when using docker-compose?

Reply:
Actually I left them as an Option to build them. But giving it a second thought It is cleaner to have them removed